.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,15 @@
 This PR/Translation is about #ISSUE_NUMBER.
 
 **files translated**:
-  - [ ] `def/--LOCALE-CODE--/about.cson`
-  - [ ] `def/--LOCALE-CODE--/context.cson`
-  - [ ] `def/--LOCALE-CODE--/menu_darwin.cson`
-  - [ ] `def/--LOCALE-CODE--/menu_linux.cson`
-  - [ ] `def/--LOCALE-CODE--/menu_win32.cson`
-  - [ ] `def/--LOCALE-CODE--/settings.cson`
-  - [ ] `def/--LOCALE-CODE--/welcome.cson`
+  - [] `def/--LOCALE-CODE--/about.cson`
+  - [] `def/--LOCALE-CODE--/context.cson`
+  - [] `def/--LOCALE-CODE--/menu_darwin.cson`
+  - [] `def/--LOCALE-CODE--/menu_linux.cson`
+  - [] `def/--LOCALE-CODE--/menu_win32.cson`
+  - [] `def/--LOCALE-CODE--/settings.cson`
+  - [] `def/--LOCALE-CODE--/welcome.cson`
 
 **validation**
-  - [ ] I've validated my translation locally by running
+  - [] I've validated my translation locally by running
```

Comment:
Please leave it `[ ]` so we can easily check/uncheck items when markdown is rendered
e.g.
`- [ ] check item`
↓↓↓
- [ ] check item

--- 
while

`- [] check item`
↓↓↓
- [] check item